src/engraving/rw/read410/read410.cpp

```diff
@@ -891,6 +894,21 @@ void Read410::pasteSymbols(XmlReader& e, ChordRest* dst)
                 b->setTrack(destTrack);
                 b->setParent(seg);
                 score->undoAddElement(b);
+            }else if(tag == "Clef") {
+                
+                Measure* meas = score->tick2measure(destTick);
+                Segment* seg = meas ? meas->undoGetSegment(SegmentType::Clef, destTick) : nullptr;
+                if (!seg) {
+                    LOGD() << "No Clef segment at tick " << destTick.ticks();
+                    e.skipCurrentElement();
+                    continue;
+                }
```

Comment:
`undoGetSegment` never returns nullptr, because if no segment of the desired type exists, it creates one (see its definition)

Reply:
Actually, never mind, about this one :)